timechop/architect.py

```diff
@@ -134,9 +134,14 @@ def build_matrix(self, as_of_times, label_name, label_type,
         """
         matrix_filename = os.path.join(
             matrix_directory,
-            '{}.csv'.format(matrix_uuid)
+            'tmp_{}.csv'.format(matrix_uuid)
         )
-        if not self.replace and os.path.exists(matrix_filename):
+        if not self.replace and os.path.exists(
+            os.path.join(
+                matrix_directory,
+                '{}.csv'.format(matrix_uuid)
+            )
+        ):
             logging.info('Skipping %s because matrix already exists', matrix_filename)
```

Comment:
Logging message no longer matches the filename that caused the skip.

I think the variable name for tmp_* should be something like `temp_matrix_filename` to reduce confusion, or keep that the same and make a var for the other one called `archive_matrix_filename` or something.